build.gradle.kts

```diff
@@ -49,7 +49,8 @@ val generateJavadocJar by tasks.registering(Jar::class) {
 
 val buildJs by tasks.registering(NpmTask::class) {
     dependsOn("npmInstall")
-    args.set(listOf("run", "buildRelease"))
+    var buildTask = if(isSnapshot()) "build" else "buildRelease"
```

Comment:
Why ? Already some regressions were uncaught on the PubSDK because artifact was tested without minification. Why tempting the same with MRAID ? Moreover, I don't believe that you can debug JS from Android/iOS anyway.

If you want to expose artifacts without minification, then what about simply:
- explaining in a DEVELOPMENT.md files that you can locally run the target `build` and get the artifact in the expected path
- export the build artifact from the CI and indicate it in DEVELOPMENT.md file

Or maybe am I missing something ?

Reply:
Okay, i will revert those changes to always minify code